WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js

```diff
@@ -289,6 +289,10 @@ Ext.define("DIRAC.JobMonitor.classes.JobMonitor", {
       JobID: {
         name: "JobID(s)",
         type: "number"
+      },
+      PilotJobReference: {
+        name: "Pilot Job Reference(s)",
+        type: "orignalText"
```

Comment:
What was meant here ?

Reply:
These lines add a new text field in the `JobMonitor` page to enter one or more pilot-job references and thus find jobs linked to these pilot-job references.

Reply:
orignalText
```suggestion
        type: "originalText"
```

Reply:
Oops I didn't see thanks
(I tested it on a VM, I didn't mount my branch on the VM and copied what I wrote in the VM on the branch, that's why there are some mistakes like this...)